### PR TITLE
Update Debian dependencies

### PR DESCRIPTION
**Pull request by uklotzde**
- Add libebur128-dev
- Add libmodplug-dev

### CODE REVIEW

**Comment by Be-ing** [comment]
I'm not sure the failure of the PPA build had anything to do with the build environment?

**Comment by uklotzde** [comment]
As the comments state this is really a "mess". I am confused about all those differing and inconsistent Debian dependencies.

**Comment by rryan** [comment]
> I'm not sure the failure of the PPA build had anything to do with the build environment?

Unfortunately it probably is, because to prepare a source package for upload to Launchpad the dependencies need to be present on the host system. I don't think they need to be in the pbuilder chroots for the PPA to succeed but I could be wrong.

File: scripts/linux-ubuntu/pbuilderrc [hunk 1]
```diff
@@ -1,6 +1,5 @@
-QT4_PACKAGES="libqt4-dev libqt4-opengl libqt4-svg libqt4-xmlpatterns libqt4-opengl-dev libqt4-sql libqt4-sql-sqlite qtkeychain-dev"
```
Comment (rryan) [inline scripts/linux-ubuntu/pbuilderrc:1]: If you remove the Qt 4 packages from the pbuilder chroots then we won't be able to build 2.1 anymore. If we never need to build it again then that's fine.

Comment (Be-ing) [reply scripts/linux-ubuntu/pbuilderrc:1]: We don't need to build 2.1 anymore.

Comment (uklotzde) [reply scripts/linux-ubuntu/pbuilderrc:1]: I was assuming that these are obsolete on this 2.3 branch. But if the file is then used to define a single builder that is used for multiple Mixxx versions this is misleading.

**Comment by uklotzde** [comment]
For comparison the [RPMFusion](https://pkgs.rpmfusion.org/cgit/free/mixxx.git/) build:
- a single .spec file
- one branch for each release

File: scripts/linux-ubuntu/pbuilderrc (hunk 1 above)
Comment (rryan) [reply scripts/linux-ubuntu/pbuilderrc:1]: These scripts are for creating / updating an Ubuntu build VM and the pbuilder chroot environments, so unfortunately there is no distinction between branches unless we made a separate VM or pbuilder chroot for each branch.

**Comment by rryan** [comment]
> For comparison the [RPMFusion](https://pkgs.rpmfusion.org/cgit/free/mixxx.git/) build:
> 
> * a single .spec file
> * one branch for each release

That seems way better. I don't know how to do that with Debian packaging but there is surely a way.

**Comment by uklotzde** [comment]
I remember some discussions about using [Open Build Service (OBS)](https://openbuildservice.org) for this purpose. Building [DEB packages](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.package_formats.html#id-1.5.6.2.7) seems to require only the common configuration files.

**Comment by uklotzde** [comment]
I have dropped the Qt4 commit.

**Pull request merged**